.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+
+on: [pull_request]
```

Comment:
I think we'll want to use `[push, pull_request]` or else `lint.yml` won't run for commits pushed after a PR is created